composer.json

```diff
@@ -78,6 +78,18 @@
         "phpactor/worse-reference-finder": "0.2.6",
         "phpactor/worse-reflection-extension": "0.2.5"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "no-api": true,
+            "url": "https://github.com/przepompownia/container"
```

Comment:
is there a PR for container also?

Reply:
https://github.com/phpactor/container/pull/12

Reply:
cool closed that, we can now just require `map-resolver: ^1.5.0`